integration-tests/bats/sql-local-remote.bats

```diff
@@ -108,3 +122,72 @@ teardown() {
     [[ "$output" =~ "defaultDB does not exist" ]] || false
 }
 
+@test "sql-local-remote: test 'status' and switch between server/no server" {
+    start_sql_server defaultDB
+
+    run dolt --verbose-engine-setup --user dolt status
```

Comment:
You should drop the --verbose-engine-setup flag since you aren't looking for the output from it (and I don't think you need to). Update three over uses below.